tooling/template-base/components/Navigation/CurrentUserNavBar.tsx

```diff
@@ -36,7 +36,7 @@ export default function CurrentUserNavBar() {
         </div>
 
         <div className="hidden md:block">
-          <button className="hidden md:block">
+          <div className="hidden md:block">
```

Comment:
Not really sure we need to duplicate this `div` with the same class names? Can probably just omit this `div` entirely.